.gitignore

```diff
@@ -60,6 +60,7 @@ cover/
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
+**/migrations/
```

Comment:
Is this what the guide suggests? If you want to ignore a `migration` folder in any subdirectory, just using `migrations` is enough. Although, it's also possible to add a separate `.gitignore` into a subfolder which scopes the effect of the entries nicely.

However, as I was looking into this, I realized that what you intend to do here is plain wrong — *do not exclude the migrations from Git*.

Moreover, I think you should _read more_ about what the migrations are, how they are used, why they exist.

Reply:
I've read the documentation. Okay, I will remove this from the `.gitignore` file. I made the mistake

Reply:
Marking as resolved.